substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/graal/snippets/StackOverflowCheckImpl.java

```diff
@@ -218,7 +199,7 @@ public void disableStackOverflowChecksForFatalError() {
          * ensures that any future calls to protectYellowZone() do not modify the stack boundary
          * again.
          */
-        yellowZoneStateTL.set(0xfefefefe);
+        yellowZoneStateTL.set(0x7EFEFEFE);
```

Comment:
Is that change because the value was negative and the guarantees failed?

Reply:
Yes.